### ISSUE TITLE
Use tracing points for JITServer code

### ISSUE THREAD

**Issue by mpirvu** [eclipse/openj9#9482]
Debugging JITServer behavior sometimes requires a verbose log which is not always available. I am proposing the addition of tracing points that should show in a Snap file in case something goes wrong.

**Comment by a7ehuo** [comment]
I will look at this one next.

**Comment by a7ehuo** [comment]
I experimented adding the trace points for JITServe (Draft PR #9681), which uses the existing trace component `j9jit`. @mpirvu Is a new trace component needed? Since JITServer is part of the JIT, it seems to make sense to just add JITServer trace points to the `j9jit` component. Below is an example of what the traces look like from JITServer. Next I'll go through the critical paths in debugging JITServer and see which ones should be added to trace points. 

```
...
20:42:03.735826000  0x0000000000017d00 j9jit.49            Event       JITServer: compThreadID=0 clientSessionData=0x7f5ca69376f0 for clientUID=6788276126833651685 seqNo=0
...
...
20:42:03.744735000  0x0000000000017d00 j9jit.50            Event       JITServer: compThreadID=0 has successfully compiled java/lang/Object.<init>()V
...
```

**Comment by mpirvu** [comment]
Yes, I think we can reuse the j9jit component for tracing.
We may need to debug the handshake between the client and the server: when connections are established, when messages are sent/received, when corner cases are encountered, etc.

**Comment by mpirvu** [comment]
Implemented in #9681
